### PR TITLE
Code fixes to build Motr rpms.

### PR DESCRIPTION
**Pull request by shashank-parulekar**
Changes in the code to build Motr rpms. Most of the changes are related to 
the function prototypes.

Signed-off-by: Shashank Parulekar <Shashank.Parulekar@seagate.com>

### CODE REVIEW

File: cob/cob.c [hunk 1]
```diff
@@ -831,8 +831,8 @@ int m0_cob_domain_destroy(struct m0_cob_domain *dom,
 	m0_buf_init(&key, &omgkey, sizeof omgkey);
 	cob_table_delete(dom->cd_fileattr_omg, tx, &key);
 
-	m0_cob_nskey_make(&nskey, &M0_COB_ROOT_FID, M0_COB_ROOT_NAME,
-			  strlen(M0_COB_ROOT_NAME));
+	rc = m0_cob_nskey_make(&nskey, &M0_COB_ROOT_FID, M0_COB_ROOT_NAME,
+			       strlen(M0_COB_ROOT_NAME));
```
Comment (NikhilBirgade) [inline cob/cob.c:835]: do you want to check for rc != 0, otherwise rc will get overwritten by line 836 m0_cob_lookup.

File: btree/btree.c [hunk 2]
```diff
@@ -11636,6 +11648,8 @@ static int ut_btree_suite_init(void)
 	m0_be_ut_seg_init(ut_seg, ut_be, BE_UT_SEG_SIZE);
 	seg = ut_seg->bus_seg;
 
+	g_process_fid = g_process_fid;
```
Comment (NikhilBirgade) [inline btree/btree.c:11651]: Is this added to avoid compilation error?

File: cob/cob.c (hunk 1 above)
Comment (shashank-parulekar) [reply cob/cob.c:835]: actually i do not need to get the return value as it was before. As a part of other fix this code remained. I will remove the assignment to rc.

File: btree/btree.c (hunk 2 above)
Comment (shashank-parulekar) [reply btree/btree.c:11651]: yes 👍

**Review by NikhilBirgade — APPROVED** [review]

**Review by ydb242 — APPROVED** [review]

**Review by RadhaGulhane13 — APPROVED** [review]

**Pull request merged**